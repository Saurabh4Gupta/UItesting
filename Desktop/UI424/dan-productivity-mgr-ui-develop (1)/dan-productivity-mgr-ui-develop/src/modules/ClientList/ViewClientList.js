import React from 'react';
import ClientList from './ClientList';
import { dataFieldCms as PageContent } from '../../cms';

const ViewClientList = (props) => <ClientList cmsData={PageContent} {...props} />;

export default ViewClientList;
