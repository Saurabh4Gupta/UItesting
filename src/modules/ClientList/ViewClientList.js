import React from 'react';
import ClientList from './ClientList';
import { dataFieldCms as PageContent } from '../../cms';

const ViewClientList = () => <ClientList cmsData={PageContent} />;

export default ViewClientList;
