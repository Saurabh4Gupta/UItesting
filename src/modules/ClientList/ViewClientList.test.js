
import React from 'react';
import { shallow } from 'enzyme';
import ViewClientList from './ViewClientList';

describe('Success test cases for: ViewClientList component', () => {
  let wrapper;

  describe('Snapshot test for  ViewClientList component', () => {
    beforeEach(() => {
      wrapper = shallow(<ViewClientList />);
    });

    it('Test to match the snapsot', () => {
      expect(wrapper).toMatchSnapshot();
    });

    it('Renders correctly', () => {
      expect(wrapper.exists()).toBe(true);
    });
  });
});

describe('Failure test cases for: Duplicate data page', () => { });
