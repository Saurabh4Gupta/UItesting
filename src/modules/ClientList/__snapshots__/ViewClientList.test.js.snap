// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success test cases for: ViewClientList component Snapshot test for  ViewClientList component Test to match the snapsot 1`] = `
<ClientList
  cmsData={
    Object {
      "actual": "actuals",
      "actualDataHint": "Select the number of months data required",
      "actualDataLabel": "Actual Productivity Data",
      "assignToHint": "Only the person or team you assign this to will have access. Once you click 'Create' ,
  they will be sent email notifications asking them to fill in the data request.",
      "assignToLabel": "Assign To",
      "briefHint": "Tell us in detail what this data request is for, max 4000 characters",
      "briefLabel": "Briefing",
      "cancel": "Cancel",
      "complete": "Complete",
      "completeLabel": "Complete",
      "create": "Create",
      "createNewDataRequest": "Create new data request",
      "datePlaceHolder": "Date",
      "delete": "Delete",
      "dueDateHint": "Enter the date that you need this data request by",
      "dueDateLabel": "Due Date",
      "edit": "Edit",
      "editDataRequest": "Edit data request",
      "emptyCompletedProductivityDatarequestCaption": "No Productivity data requests have been moved to this section yet.",
      "emptyLabel": "Create new data request",
      "emptyProductivityDatarequestCaption": "There are currently no active Productivity data requests",
      "forecast": "forecast",
      "forecastDataHint": "Select the number of months data required",
      "forecastDataLabel": "Forecast Productivity Data",
      "lastUpdate": "Last Update",
      "localMarketHint": "Choose the market you wish to create this data request for",
      "localMarketLabel": "Local Market",
      "moveToComplete": "Move to Complete",
      "nameHint": "Make sure it's clear to help others identify it.Max 255 characters",
      "nameLabel": "Data request name",
      "ongoing": "Ongoing",
      "ongoingLabel": "Ongoing",
      "overdue": "Overdue",
      "productivityDatarequestCaption": " The ongoing and complete productivity data requests created for this client",
      "productivityDatarequestHeading": "Productivity data requests",
      "selectPlaceHolder": "Please Select",
      "templateFileHint": "Only .xls, .xlsx, .csv, .xlsb, .xlsm, .xlt, .xltm, .xltx files in 20mb or less",
      "templateFileLabel": "Add tracker template file",
      "uploadButtonText": "Upload new file",
      "uploadFileLabel": "To upload a new file,drag and drop you file below or browse using the link",
      "viewClientsCaption": "Choose the client you wish to view",
      "viewClientsHeading": "My clients",
      "viewDetails": "View Details",
    }
  }
  history={Object {}}
/>
`;
